src/components/About/assets/content.json

```diff
@@ -0,0 +1,15 @@
+{
+  "project": {
+    "repo": {
+      "client": "https://github.com/Scott-Kenning/StudySpaceFinder",
```

Comment:
this might be more readable if it's put in consts.js, and then split in to several variables so we don't have to do things like "content.project.contributors"

Reply:
Oh true, will do.

Reply:
fixed it in latest commit